cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/ForecastInstance.java

```diff
@@ -355,18 +357,19 @@ public boolean equals(Object o) {
                 && Objects.equals(this.firstDateTime, forecastInstance.firstDateTime)
                 && Objects.equals(this.lastDateTime, forecastInstance.lastDateTime)
                 && Objects.equals(this.maxAge, forecastInstance.maxAge)
-                && Objects.equals(this.timeSeriesCount, forecastInstance.timeSeriesCount)
                 && this.notes == null || forecastInstance.notes == null ? Objects.equals(this.notes, forecastInstance.notes) : this.notes.equalsIgnoreCase(forecastInstance.notes)
                 && Objects.equals(this.metadata, forecastInstance.metadata)
                 && this.filename == null || forecastInstance.filename == null ? Objects.equals(this.filename, forecastInstance.filename) : this.filename.equalsIgnoreCase(forecastInstance.filename)
                 && this.fileDescription == null || forecastInstance.fileDescription == null ? Objects.equals(this.fileDescription, forecastInstance.fileDescription) : this.fileDescription.equalsIgnoreCase(forecastInstance.fileDescription)
+                && this.fileMediaType == null || forecastInstance.fileMediaType == null ? Objects.equals(this.fileMediaType, forecastInstance.fileMediaType) : this.fileMediaType.equalsIgnoreCase(forecastInstance.fileMediaType)
                 && Arrays.equals(this.fileData, forecastInstance.fileData)
-                && this.fileDataUrl == null || forecastInstance.fileDataUrl == null ? Objects.equals(this.fileDataUrl, forecastInstance.fileDataUrl) : this.fileDataUrl.equalsIgnoreCase(forecastInstance.fileDataUrl);
+                && this.fileDataUrl == null || forecastInstance.fileDataUrl == null ? Objects.equals(this.fileDataUrl, forecastInstance.fileDataUrl) : this.fileDataUrl.equalsIgnoreCase(forecastInstance.fileDataUrl)
+                ;
```

Comment:
How hard would it be to fix the codegen to not put this semicolon on the next line? 

Not a big deal, but noticed it always does that.